hasura/hasura-allow-operations-in/src/cli.ts

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+import { run } from './run';
+
+const sourcePath = process.argv[2];
+const hasuraUri = process.argv[3];
+const allowIntrospection = process.argv[4] === 'true';
+
+if (sourcePath === undefined) {
+  throw new Error('Source path must be passed as first argument');
+}
+
+if (hasuraUri === undefined) {
+  throw new Error('Hasura URI must be passed as the second argument');
+}
```

Comment:
Any reason why this one is passed as an argument here whereas it's passed as an ENV var everywhere else?

Reply:
I was going to annotate this to explain. I've implemented a standalone utility that is useful outside the context of Cardano GraphQL, and possibly even worth lifting to another repo (can handle that later though). In this context all arguments are provided in the command.

The ENV is an implementation detail from Cardano GraphQL, and is mapped through when the [package is implemented](https://github.com/input-output-hk/cardano-graphql/blob/feature/hasura-allow-operations-in/src/util/hasuraAllowOperationsIn.ts#L21)